src/scripts/site.js

```diff
@@ -1012,7 +969,7 @@ var AppInfo = {};
                 }
 
                 if ("livetv" === item) {
-                    if ("programs" === options.section) {
+                    if ("progra        var bowerPath = getBowerPath();" === options.section) {
```

Comment:
hmmm

Reply:
Good catch. That damn middle click paste... It should be disabled, but somehow isn't...